client/src/pages/login/judgingDash/judging.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './judging.css';
+
+function Judging() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Ignore this